truthy-falsy.js

```diff
@@ -0,0 +1,219 @@
+  return opponent === 'scissors' || 'rock';
+}
+
+function rockPaperScissorsLizardSpock(playerOne, playerTwo) {
+  if (playerOne === playerTwo) {
+    return 'Draw!';
+  }
+  if (playerOne === 'rock' && willRockWin(playerTwo)) {
+    return 'Player 1 Won!';
+  }
+  if (playerOne === 'paper' && willPaperWin(playerTwo)) {
+    return 'Player 1 Won!';
+  }
+  if (playerOne === 'scissors' && willScissorsWin(playerTwo)) {
+    return 'Player 1 Won!';
+  }
+  if (playerOne === 'lizard' && willLizardWin(playerTwo)) {
+    return 'Player 1 Won!';
+  }
+  if (playerOne === 'spock' && willSpockWin(playerTwo)) {
+    return 'Player 1 Won!';
+  }
+  return 'Player 2 Won!';
+}
+
+// 8
+function isValidPassword(password) {
+  const specialCharacters = /[`!@#$%^&*()_\-+=\[\]{};':"\\|,.<>\/?~ ]/;
+  const passwordLengthCheck = password.trim().length >= 8;
+  const passwordCapitalCharactersCheck = /[A-Z]/.test(password) >= 1;
+  const passwordSpecialCharactersCheck = specialCharacters.test(password) >= 1;
+
+  if (passwordLengthCheck && passwordCapitalCharactersCheck && passwordSpecialCharactersCheck) {
+    return true;
+  }
+  return false;
+}
+console.log(isValidPassword('passSword!'));
+
+// 9
+function isPasswordLengthValid(password) {
+  if (password.trim().length >= 8) {
+    return 1;
+  }
+  return 0;
+}
+console.log(isPasswordLengthValid('pass'));
+
+function hasUppercaseLetter(password) {
+  if (/[a-z]/.test(password) >= 1 ) {
+    return 1;
+  }
+  return 0;
+}
+
+function hasLowercaseLetter(password) {
+  if (/[A-Z]/.test(password) >= 1 ) {
+    return 1;
+  } return 0;
+}
+
+function hasSpecialLetter(password) {
+  const specialChars =/[`!@#$%^&*()_\-+=\[\]{};':"\\|,.<>\/?~ ]/;
+  if (specialChars.test(password) >= 1 ) {
+    return 1;
+  }
+  return 0;
+}
+
+function getPasswordRating(password) {
+  const passwordLength = isPasswordLengthValid(password);
+  const passwordLowercase= hasUppercaseLetter(password);
+  const passwordUppercase = hasLowercaseLetter(password);
+  const passwordSpecialLetter = hasSpecialLetter(password);
+  return passwordLength + passwordLowercase + passwordUppercase + passwordSpecialLetter;
+}
+console.log(getPasswordRating('passWord'));
+
+// 10
+
+function getVipDiscount(isVipMember) {
+  if (isVipMember) {
+    return 0.05;
+  }
+  return 1;
+}
+
+function getLoyaltyDiscount(loyaltyPoints) {
+  if (loyaltyPoints) {
+    return loyaltyPoints * 0.01;
+  }
+  return 0;
+}
+
+function getInternationalShippingPrice(isShippedInternationally) {
+  if (isShippedInternationally) {
+    return 10;
+  }
+  return 5;
+}
+
+function getTotalOrderCost (baseItemCost, isVipMember, loyaltyPoints, isShippedInternationally) {
+  const vipMemberDiscount = getVipDiscount(isVipMember);
+  const loyaltyDiscount = getLoyaltyDiscount(loyaltyPoints);
+  const shippingPrice = getInternationalShippingPrice(isShippedInternationally);
+
+  return baseItemCost - (baseItemCost * vipMemberDiscount) - loyaltyDiscount + shippingPrice
```

Comment:
You missed a semicolon here. Please try not to make the same mistakes multiple times